src/app/screens/Login/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useCallback } from 'react';
+import i18next from 'i18next';
+import PropTypes from 'prop-types';
+import { useToasts } from 'react-toast-notifications';
+import clsx from 'clsx';
+
+import useAPI from '~hooks/useAPI';
+import Loader from '~components/Loader';
+import PublicLayoutWrapper from '~components/PublicLayoutWrapper';
+import { ENDPOINTS } from '~constants/api';
+import wolox from '~assets/logos/wolox.png';
+
+import Form from './components/Form';
+import styles from './styles.module.scss';
+
+export default function Login({ history }) {
+  const { addToast } = useToasts();
+  const [{ isLoading, isError, response }, doFetch] = useAPI(
+    {
+      url: ENDPOINTS.login,
+      method: 'POST'
+    },
+    null
+  );
+
+  const handleSignUp = useCallback(() => {
+    // eslint-disable-next-line no-console
+    console.log('se logueo con exito', response);
+  }, [response]);
+
+  useEffect(() => {
+    if (response) {
+      if (isError) {
+        addToast(response, { appearance: 'error' });
```

Comment:
sorry for this, but could yo extract the. appearances in an object global constant?

```
addToast(response, TOAST_TYPES.error);
```